### PR TITLE
Add Check to AMA Log Collector

### PR DESCRIPTION
**Pull request by aswatt**
Adds check to run `du -h -d 1` on AMA events folder. Also cleans up subprocess module import, as it isn't used.

### CODE REVIEW

File: AzureMonitorAgent/ama_tst/modules/logcollector.py [hunk 1]
```diff
@@ -234,6 +234,18 @@ def create_outfile(output_dirpath, logs_date, pkg_manager):
             outfile.write("--------------------------------------------------------------------------------\n")
         outfile.write("--------------------------------------------------------------------------------\n")
 
+        # du output on events folder
+        for flag in ["", "--apparent-size"]:
+            du_full_cmd = DU_CMD.format(flag)
+            outfile.write("Output of command: {0}\n".format(du_full_cmd))
+            outfile.write("========================================\n")
+            outfile.write(helpers.run_cmd_output(du_full_cmd))
+            outfile.write("--------------------------------------------------------------------------------\n")
```
Comment (AaronRM) [inline AzureMonitorAgent/ama_tst/modules/logcollector.py:243]: Why not match the starting line format ======= ?

File: AzureMonitorAgent/ama_tst/modules/logcollector.py [hunk 2]
```diff
@@ -234,6 +234,18 @@ def create_outfile(output_dirpath, logs_date, pkg_manager):
             outfile.write("--------------------------------------------------------------------------------\n")
         outfile.write("--------------------------------------------------------------------------------\n")
 
+        # du output on events folder
+        for flag in ["", "--apparent-size"]:
+            du_full_cmd = DU_CMD.format(flag)
+            outfile.write("Output of command: {0}\n".format(du_full_cmd))
+            outfile.write("========================================\n")
+            outfile.write(helpers.run_cmd_output(du_full_cmd))
```
Comment (AaronRM) [inline AzureMonitorAgent/ama_tst/modules/logcollector.py:242]: Do we need to append a newline after the cmd output?

**Review by AaronRM — APPROVED** [review]

File: AzureMonitorAgent/ama_tst/modules/logcollector.py (hunk 1 above)
Comment (aswatt) [reply AzureMonitorAgent/ama_tst/modules/logcollector.py:243]: The `=` line separates the command and its output, while the `-` line separates different commands; example below:

```
Output of command: ls -l
========================================
-rw-rw-r-- 1 aswatt aswatt 21275 Apr  4 16:22 ama_tst.tgz
--------------------------------------------------------------------------------
Output of command: pwd
========================================
/home/aswatt
--------------------------------------------------------------------------------```

I was trying to make it the most legible, but lmk if you think there's a better way to format this!

File: AzureMonitorAgent/ama_tst/modules/logcollector.py (hunk 2 above)
Comment (aswatt) [reply AzureMonitorAgent/ama_tst/modules/logcollector.py:242]: The `run_cmd_output()` function automatically adds a newline already

**Pull request merged**